scripts/CEE/cluster-health-check/metadata.yaml

```diff
@@ -0,0 +1,95 @@
+file: run-oc-hc.sh
+name: cluster-health-check
+description: Check the overall health of an OpenShift cluster
+author: Givaldo Lins
+allowedGroups:
+  - CEE
+  - SREP
+  - CSSRE
+  - LPSRE
+  - MTSRE
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - get
+        - list
+        - watch
+      apiGroups:
+        - ''
+      resources:
+        - namespaces
+    - verbs:
+        - get
+        - list
+        - watch
+        - patch
```

Comment:
What is the check doing that it needs write access to `alertmanagers`?

Reply:
It lists all firing alerts in all namespaces. I was only able to get the alerts once the patch verb was added, but it only reads them, no change is made.
https://github.com/givaldolins/openshift-cluster-health-check/blob/main/oc-hc/cmd/alertsStatus.go#L54-L71

Reply:
There's a specific non-existent resource you can provide access to authorize to alertmanager without requiring write access to existing resources: https://github.com/openshift/cluster-monitoring-operator/blob/cd076c6741e7928741e8b8528e97804fff399e4e/assets/alertmanager/alertmanager.yaml#L32

Reply:
That worked. I have updated the PR

Reply:
I am still seeing the 'patch' verb there.

Reply:
That is needed, otherwise the alerts are not listed. But as @cblecker suggested, it has been limited to the non-existant resource https://github.com/openshift/managed-scripts/blob/98dc857a825db5544fa51ce09279ac51aacde515/scripts/CEE/cluster-health-check/metadata.yaml#L29-L36